### PR TITLE
added support fo file printing

### PR DESCRIPTION
**Pull request by Royvaknin29**

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -63,6 +63,7 @@ export const validators: {
 export interface ValidatorOptions {
     apiDefinitionsPath: string;
     reportCoverage?: boolean;
+    printCoverageToFile?: boolean;
```
Comment (kobik) [inline types/index.d.ts]: lets change `printCoverageToFile` to `exportCoverage`. we also need to document it in the readme

**Comment by kobik** [comment]
all builds are failing on lint validation

File: lib/helpers/coverage.js [hunk 2]
```diff
@@ -9,73 +9,74 @@ const buildTable = require('./coverage-table');
...
-    }
-    if (printCoverageToFile === true) {
-        process.on('beforeExit', () => {
-            const parsedCoverage = getReport();
-            printReportToFile(parsedCoverage);
-        });
-    }
+  if (reportCoverage === true) {
+    process.on('beforeExit', () => {
+      const parsedCoverage = getReport();
```
Comment (kobik) [inline lib/helpers/coverage.js]: add another event listener for getting the report once

File: README.md [hunk 3]
```diff
@@ -133,6 +133,23 @@ Uncovered API definitions found:
 /v2/pet/:petId/uploadImage | POST       | 200         
 ```
 
+# Exporting the report: 
+When providing `reportCoverage:true`, `exportCoverage: true` a `coverage.json` file will be created in your cwd with following structure:
```
Comment (kobik) [inline README.md]: description is a little confusing as it sounds that both are required in order to export the report

File: types/index.d.ts (hunk 1 above)
Comment (Royvaknin29) [reply types/index.d.ts]: done

File: lib/helpers/coverage.js (hunk 2 above)
Comment (Royvaknin29) [reply lib/helpers/coverage.js]: done

File: README.md (hunk 3 above)
Comment (Royvaknin29) [reply README.md]: done

File: coverage.json [hunk 4]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (kobik) [inline coverage.json]: delete unnecessary file

**Pull request merged**